### ISSUE TITLE
AckeeApiError: `GRAPHQL_VALIDATION_FAILED`

### ISSUE THREAD

**Issue by ilya-vasilyev** [BetaHuhn/ackee-report#88]
**🐞 Describe the bug**

When running Ackee Report I get this error:

```
✖  AckeeApiError: {"errors":[{"message":"Field \"averageViews\" of type \"AverageViews!\" must have a selection of subfields. Did you mean \"averageViews { ... }\"?","locations":[{"line":7,"column":8}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED"}},{"message":"Field \"averageDuration\" of type \"AverageDuration!\" must have a selection of subfields. Did you mean \"averageDuration { ... }\"?","locations":[{"line":8,"column":8}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED"}}]} (400 status code)
```

**📚 To Reproduce**

- Install ackee-report
- Configure token and server
- Try to do `ackee-report html -d all` 

**💡 Expected behavior**

Report to be generated

**🖼️ Screenshots**

![image](https://user-images.githubusercontent.com/10021219/123918281-63ec2000-d98c-11eb-8d39-90def5a2114b.png)

**⚙️ Environment**
 - OS: **Linux Mint 20.1**
 - Shell **fish**
 - Ackee version **3.1.1**
 - Ackee Report version **1.3.3**

**Comment by BetaHuhn** [comment]
This error is likely occurring because of the recent changes to Ackee‘s GraphQL API in [v3.1.0](https://github.com/electerious/Ackee/releases/tag/v3.1.0).

Will fix this later today.

**Comment by BetaHuhn** [comment]
Fixed in [v2.0.0](https://github.com/BetaHuhn/ackee-report/releases/tag/v2.0.0).

`ackee-report@latest` now requires Ackee >=v3.1.0.
